deploy/environments/applications.yaml

```diff
@@ -41,7 +41,7 @@ spack:
     - neurodamus-thalamus+coreneuron%intel^coreneuron+knl
     - parquet-converters
     - placement-algorithm
-    - psp-validation
+    - psp-validation%gcc ^neuron%intel
```

Comment:
How about a safeguard for neuron, too? `conflicts('%gcc')`? Is that too much of a sledgehammer?

Like this, you will have to manually ensure that all software has the little `^` behind it. Does not seem wise…

Reply:
@matz-e : this applies to BB5 only. While using Spack locally, I would still like to build with compiler like GCC.